packages/@dcl/inspector/src/lib/data-layer/host/rpc-methods.ts

```diff
@@ -24,7 +27,21 @@ function setupEngineDump(
     if (!dump) return composite
 
     const mainCrdt = dumpEngineToCrdtCommands(engine)
-    fs.writeFile('main.crdt', Buffer.from(mainCrdt)).catch((err) => console.error(`Failed saving main.crdt: `, err))
+    fs.writeFile('main.crdt', Buffer.from(mainCrdt)).catch((err) => console.error('Failed saving main.crdt: ', err))
+
+    fs.readFile('scene.json')
```

Comment:
do we need to do this on every engine.update ?
Can we attach it only if the SceneComponent is dirty or if the onChange cb ?

Reply:
done :D